gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -80,7 +80,12 @@ class BaselineIdea extends AbstractBaselinePlugin {
             }
         }
 
-        project.getTasks().named("idea").configure(idea -> idea.doLast(cleanup))
+        project.getTasks().named("idea").configure(idea -> {
+            idea.doFirst(_t -> project.getLogger().warn("DEPRECATED: Using `./gradlew idea` is no longer recommended, some " +
+                    "functionality may not function as expected. Instead, we suggest opening the project directly in " +
+                    "IntelliJ or running `idea .` (if the JetBrains utility is installed) for a smoother experience."))
```

Comment:
I think we need to tell people exactly how to get out of their situation, which is to:

1. Close the intellij project
2. Run `./gradlew cleanIdea`
3. Run `rm -rf .idea || true`
4. Open project in IntelliJ or use `idea .` if installed by JetBrains Toolbox.

Maybe say they only need to do step 4 for new projects?

Reply:
updated in https://github.com/palantir/gradle-baseline/pull/2711/commits/2d7e707d279332ad6250f006859f9630337a2c8f